Adamant/Stories/Settings/Contribute/ContributeState.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  ContributeState.swift
+//  Adamant
+//
+//  Created by Stanislav Jelezoglo on 14.06.2023.
+//  Copyright © 2023 Adamant. All rights reserved.
+//
+
+import SwiftUI
+
+struct ContributeState: Equatable {
+    var name: Text
```

Comment:
`String`
Text is SwiftUI entity, view model should be independent from specific frameworks